nss-day-cohort-44/frosty-flakes-rare-client
# change to fetch call for my posts

# Description
New fetch call for myposts, updated UserPostList 

## Issue ticket: #4 

## Type of change
- [x ] New feature (non-breaking change which adds functionality)

# Testing Instructions
`git fetch origin jl-myposts`
`git checkout jl-myposts `
`npm start`

Pull down both client and server jl-myposts - check that when you click on myposts in react app, you are given the current users posts

`git branch -d jl-myposts `

# Checklist:
- [ x] I have performed a self-review of my own code
- [ ] I have commented my code with my name at the top, and in particularly hard-to-understand areas
- [x ] My changes generate no errors
- [ x] I have added test instructions that prove my fix is effective or that my feature works
